sources/packages/api/src-sql/Student/Add-sin-consent-col.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS sin_consent BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Should this one have a default value?

Reply:
If I remove the default value, should we allow null in that column ?

Reply:
If we assume that it is a mandatory column upon student creation it can be `not null`.

Reply:
Maybe just drop the constraint as @dheepak-aot mentioned here https://github.com/bcgov/SIMS/pull/1417#discussion_r977908902

Reply:
Dropped.